gen_cert.py

```diff
@@ -131,7 +131,7 @@ def __init__(self, course_id, template_pdf=None, aws_id=None, aws_key=None,
 
         def interstitial_factory():
             """ Generate default values for interstitial_texts defaultdict """
-            return itertools.repeat(cert_data.get('interstitial', {}).get('Pass', '')).next
+            return itertools.repeat(cert_data.get('interstitial', {}).get('Pass', '').encode('utf-8')).next
```

Comment:
We probably want to be using unicode strings everywhere, instead of relying on them being converted from ASCII later; that's what caused this class of bug in the first place. Unless we explicitly want ASCII, we should always keep everything in unicode.